### PR TITLE
Input solved and fixed sidebar menu in mobile

### PR DESCRIPTION
**Pull request by israelperezglez**
@dialelo @raulraja could you review please?

- We delete the border-bottom for input solved
- We change all style for menu exercises in mobile

captura de pantalla 2016-07-01 a las 13 41 25

### CODE REVIEW

File: server/app/assets/stylesheets/modules/_navbar.scss [hunk 1]
```diff
@@ -54,4 +55,76 @@
...
+            &:focus {
+                color: $gray-dark;
+            }
+        }
+        @media (max-width: $grid-float-breakpoint-max) {
+            // Dropdowns get custom display
+            .open .dropdown-menu {
+                > li > a {
+                    color: $gray-dark;
+                    ;
```
Comment (dialelo) [inline server/app/assets/stylesheets/modules/_navbar.scss]: This semicolon (`;`) is needed? There are a few floating around.

**Comment by israelperezglez** [comment]
I updated _navbar.scss

**Comment by dialelo** [comment]
Cool, taking a look to the changes.

**Comment by dialelo** [comment]
LGTM :+1:

**Pull request merged**